<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>652</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>651</height>
     </rect>
    </property>
    <widget class="QWidget" name="home">
     <widget class="QPushButton" name="btnAlgo">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Algorithmes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>291</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BIENVENUE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSaisie">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>390</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Saisie de graphe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCharger">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>300</y>
        <width>351</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Charger un graphe</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageDantzig2">
     <widget class="QPushButton" name="btnRetourDantzig_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsViewDantzig_2">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>90</y>
        <width>501</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelMatDantzig1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelMatDantzig2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>330</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageDantzig">
     <widget class="QPushButton" name="btnRetourDantzig">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExecDantzig">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>580</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Exécuter l'algorithme</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsViewDantzig">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>881</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageAlgos">
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>381</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ALGORITHMES</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnKruskal">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>460</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kruskal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDistances">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>100</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Distances</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRang">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rang</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTarjan">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tarjan</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOrdonnancement">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ordonnancement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPrufer">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>520</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Prüfer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDjikstra">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>340</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Djikstra</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnDantzig">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>400</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Dantzig</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRetourAlgos">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageKruskal">
     <widget class="QPushButton" name="btnRetourKruskal">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsViewKruskal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>881</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExecKruskal">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>580</y>
        <width>281</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Exécuter l'algorithme</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageKruskal2">
     <widget class="QPushButton" name="btnRetourKruskal2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelAretesKruskal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>501</width>
        <height>481</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsViewKruskal2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>100</y>
        <width>431</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageChargerGraphe">
     <widget class="QPushButton" name="btnRetourChargerGraphe">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>571</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CHARGER UN GRAPHE</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFichiers">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>851</width>
        <height>391</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="editTxtFichier2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>570</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom du fichier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnChoisirGrapheCourant">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>570</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Choisir comme graphe courant</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageSaisie">
     <widget class="QPushButton" name="btnRetourSaisie">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAjouter">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>400</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>481</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SAISIE DES ARETES</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Du sommet</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>au sommet</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSommet1">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>180</y>
        <width>71</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtSommet2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>180</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxOriente">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>330</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>320</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Orienté</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBoxValue">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>330</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>320</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Valué</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnTerminer">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>480</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Terminer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtValeurArete">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>250</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Valeur de l'arête</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="pageGraphe">
     <widget class="QPushButton" name="btnPageAccueil">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Page d'accueil</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>891</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSave">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>570</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Sauvegarder le graphe</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelFS">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>40</y>
        <width>691</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelAPS">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>691</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="labelAdj">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>251</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="editTxtFichier1">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>570</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>570</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom du fichier</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
